src/utils.ts

```diff
@@ -128,9 +128,10 @@ export function getApikey(url: string) {
 }
 export function cleanseSeparators(str: string): string {
 	return str
-		.replace(/\[.*?\]|「.*?」|｢.*?｣|【.*?】/g, "")
-		.replace(/[._()[\]]/g, " ")
-		.replace(/\s+/g, " ")
+		.replace(/\[.*?\]|「.*?」|｢.*?｣|【.*?】/g, "") // bracketed text
+		.replace(/[._()[\]]/g, " ") // release delimiters (except '-')
+		.replace(/\s+/g, " ") // successive spaces
+		.replace(/^-+|-+$/g, "") // "trim()" hyphens
```

Comment:
Should we cover white spaces? It might appear since we are removing stuff so I think it's necessary. Otherwise we would need to call trim().

```suggestion
		.replace(/^\s*-+s\*|\s*-+\s*$/g, "") // "trim()" hyphens
```

Reply:
I think assuming that this will always be surrounded by spaces is flawed. I think the following is potentially a better option that the end `trim()` addresses any space remnants that would remain?

`^\s*?-+|-+\s*?$`

Reply:
The * covers the zero case.

Reply:
https://regex101.com/r/654OSp/1

Test case

Reply:
Sorry, sick and just woke up. You're correct.

So `^\s*-+|-+\s*$`

https://regex101.com/r/654OSp/2

Reply:
Yeah that works, I forgot about trim() so both sides aren't necessary. I had a typo in my original so that could have been the issue.

Reply:
I've updated that, if you can resolve this and approve if it looks good to you.